mdata/aggmetric_test.go

```diff
@@ -17,6 +19,12 @@ type point struct {
 	val float64
 }
 
+type points []point
+
+func (a points) Len() int           { return len(a) }
+func (a points) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+func (a points) Less(i, j int) bool { return a[i].ts < a[j].ts }
+
```

Comment:
the naming convention in golang seems to be that a type whose purpose is to enable sorting, is that that type is named after how the sorting is done.  for example see the `ByAge` type at the first example at https://golang.org/pkg/sort/ 
So this would be called ByTs and then the call will look nicer (`sort.Sort(ByTs(c.points))`)